db/migrate/20180209163253_split_post_text_into_replies.rb

```diff
@@ -0,0 +1,59 @@
+class SplitPostTextIntoReplies < ActiveRecord::Migration[5.1]
+  def up
+    execute <<~SQL
+      UPDATE replies
+      SET reply_order = reply_order + 1;
+    SQL
+
+    Post.find_each do |post|
+      reply = Reply.create!(
+        post_id: post.id,
+        user_id: post.user_id,
+        character_id: post.character_id,
+        character_alias_id: post.character_alias_id,
+        icon_id: post.icon_id,
+        content: post.content,
+        created_at: post.created_at,
+        updated_at: post.updated_at,
+        skip_post_update: true
+      )
+      reply.update_columns(reply_order: 0, created_at: post.created_at, updated_at: post.edited_at)
```

Comment:
This... is going to take a long time. I don't know if there's any way to improve that.